rust-runtime/aws-smithy-runtime/src/client/retries/strategy/fixed_delay.rs

```diff
@@ -72,31 +73,29 @@ impl RetryStrategy for FixedDelayRetryStrategy {
             return Ok(ShouldAttempt::No);
         }
 
-        let retry_classifiers = runtime_components
-            .retry_classifiers()
-            .expect("a retry classifier is set");
-        let retry_reason = retry_classifiers.classify_retry(ctx);
+        let retry_classifiers = runtime_components.retry_classifiers();
+        let classifier_result = run_classifiers_on_ctx(retry_classifiers, ctx);
 
-        let backoff = match retry_reason {
-            Some(RetryReason::Explicit(_)) => self.fixed_delay,
-            Some(RetryReason::Error(_)) => self.fixed_delay,
-            Some(_) => {
-                unreachable!("RetryReason is non-exhaustive. Therefore, we need to cover this unreachable case.")
-            }
-            None => {
-                tracing::trace!(
+        let backoff = match classifier_result {
+            RetryAction::RetryAfter(_) => self.fixed_delay,
```

Comment:
Shouldn't this use the delay inside of `RetryAfter`? Is there a test for explicit retry delay?

Reply:
It's up for debate imo. If you set a retry strategy that's meant to always use the same delay, then I think no. I think we could just remove this strategy now. I believe I originally created it before we had Russell's `InstantSleep`. I'll remove it in a separate PR.

Reply:
nvm, I just removed it in this one since nothing was using it.